README.md

```diff
@@ -200,6 +208,11 @@ PrivateKey senderPrivateKey = senderSeed.derivePrivateKey();
 PublicKey senderPublicKey = senderPrivateKey.derivePublicKey();
 Address senderAddress = senderPublicKey.deriveAddress();
 
+KeyPair pair = Seed.ed25519Seed().deriveKeyPair(); // <-- Generate Key Pair from random seed;
+PublicKey pubKey = pair.publicKey(); // <-- Public key from KeyPair
+PrivateKey privKey = pair.privateKey() // <-- Private key from KeyPair
+Address classicAddress = pubKey.deriveAddress();
```

Comment:
I think these lines are duplicative of the lines above. I'd rather fix the lines of code above line so:

```java
// Construct a SignatureService that uses in-memory Keys (see SignatureService.java for alternatives).
SignatureService signatureService = new BcSignatureService();

// Sender (using ed25519 key)
Seed senderSeed = Seed.ed255519Seed();
KeyPair senderKeyPair = senderSeed.deriveKeyPair();
PrivateKey senderPrivateKey = senderKeyPair.privateKey();
PublicKey senderPublicKey = senderPrivateKey.publicKey();
Address senderAddress = senderPublicKey.deriveAddress();
```

Reply:
@nkramer44  we are using 3.2.1 for the lib and i don't see `ed255519Seed ` and even `org.xrpl.xrpl4j.crypto.keys.Seed` use this package, are we missing something ? crypto.core doesn't exist

Reply:
My guess is you likely have an old version of xrpl4j-crypto-core on your classpath. After v3.0, you should only have `xrpl4j-core` and `xrpl4j-client` on your classpath. See [this section of the README](https://github.com/XRPLF/xrpl4j#maven-installation) for an example of how to setup xrpl4j with maven.

Reply:
@Try-Parser I see you resolved this comment, but I'd still like to fix lines 206-209 instead of adding these lines.

Reply:
hi @nkramer44 is it better let them see how to extract PrivateKey and PublicKey from the KeyPair since they would need to extract it when they want to save it in some database ?

Reply:
I agree -- I think your code you added is correct, I just want to replace lines 205-209 with the code you added here. #493 fixes this issue